src/components/PostDetails.tsx

```diff
@@ -1,118 +1,97 @@
+  const [postComments, setPostComments] = useState<Comment[]>([]);
+  const [isLoaderPost, setIsLoaderPost] = useState(false);
 
-          <p className="title is-4">Comments:</p>
+  useEffect(() => {
+    setIsLoaderPost(true);
+    getComments(postSelectedId)
+      .then(comments => {
+        setPostComments(comments);
+        setIsLoaderPost(false);
+      });
+  }, [postSelectedId]);
 
-          <article className="message is-small" data-cy="Comment">
-            <div className="message-header">
-              <a href="mailto:misha@mate.academy" data-cy="CommentAuthor">
-                Misha Hrynko
-              </a>
-              <button
-                data-cy="CommentDelete"
-                type="button"
-                className="delete is-small"
-                aria-label="delete"
-              >
-                delete button
-              </button>
-            </div>
+  const openFormHandler = () => {
+    setIsShowForm(true);
+  };
 
-            <div className="message-body" data-cy="CommentBody">
-              Some comment
-            </div>
-          </article>
+  const deleteComment = (comment: Comment) => {
+    if (!comment.id) {
+      return;
+    }
 
-          <article className="message is-small" data-cy="Comment">
-            <div className="message-header">
-              <a
-                href="mailto:misha@mate.academy"
-                data-cy="CommentAuthor"
-              >
-                Misha Hrynko
-              </a>
+    removeComment(comment.id);
 
-              <button
-                data-cy="CommentDelete"
-                type="button"
-                className="delete is-small"
-                aria-label="delete"
-              >
-                delete button
-              </button>
-            </div>
-            <div
-              className="message-body"
-              data-cy="CommentBody"
-            >
-              One more comment
-            </div>
-          </article>
+    setPostComments(prevState => (
+      prevState.filter(c => c.id !== comment.id)
+    ));
+  };
 
-          <article className="message is-small" data-cy="Comment">
-            <div className="message-header">
-              <a
-                href="mailto:misha@mate.academy"
-                data-cy="CommentAuthor"
-              >
-                Misha Hrynko
-              </a>
+  return (
+    <>
+      {showSideBar && (
+        <div className="content" data-cy="PostDetails">
+          <div className="block">
+            <h2 data-cy="PostTitle">
+              {`#${selectedPost?.id}: ${selectedPost?.title}`}
+            </h2>
+            ≠
+            <p data-cy="PostBody">
+              {selectedPost?.body}
+            </p>
+          </div>
+          <div className="block">
+            {isLoaderPost ? <Loader /> : (
+              <CommentList
+                postComments={postComments}
+                deleteComment={deleteComment}
+              />
+            )}
```

Comment:
This syntax looks better in my opinion 
```suggestion
            {isLoaderPost ? ( 
              <Loader /> 
            ) : (
              <CommentList
                postComments={postComments}
                deleteComment={deleteComment}
              />
            )}
```